core/src/main/kotlin/transformers/documentation/PreMergeDocumentableTransformer.kt

```diff
@@ -4,5 +4,5 @@ import org.jetbrains.dokka.model.DModule
 import org.jetbrains.dokka.plugability.DokkaContext
 
 interface PreMergeDocumentableTransformer {
-    operator fun invoke(modules: List<DModule>): List<DModule>
+    operator fun invoke(modules: List<DModule>, context: DokkaContext): List<DModule>
```

Comment:
This change was necessary while I tried implementing the reporting mechanism using a `PreMergeDocumentableTransformer`. I think this might make sense for the public API. Should I revert?

Reply:
Actually you can pass context/logger to any extension point, including `PreMergeDocumentableTransformer` while registering. Check out how it's done with `ModuleAndPackageDocumentationTransformer` which previously had been provided with context via constructor.

Reply:
See comment about: changes reverted ☺️